core/frontend/src/tile/PrimaryTileTree.ts

```diff
@@ -198,11 +228,13 @@ class PrimaryTreeReference extends TileTreeReference {
   public get treeOwner(): TileTreeOwner {
     const newId = this.createTreeId(this.view, this._id.modelId, this._id.treeId.animationTransformNodeId);
     if (0 !== compareIModelTileTreeIds(newId, this._id.treeId)) {

```

Comment:
If the only thing that changed is this._forceNoInstancing then you're not going to update your Id.
The viewport already invalidates the scene when the display transform provider changes so you don't have to worry about that - you just need to make sure you compute a correct tree Id.

Reply:
Check it where, in the listener and then if it changed create a new id?  How does that make the tiles reload?

Reply:
Right here where you are checking compareIModelTileTreeIds.
TileTreeReference.treeOwner is called whenever we want access to the reference's TileTree. We compute an Id here based on the reference's current state, and ask for that tree. If the reference's state changed, we may get back a different tree - e.g., one with instancing disabled.